base/cvd/cuttlefish/host/commands/cvd/fetch/fetch_cvd.cc

```diff
@@ -324,27 +325,33 @@ Result<std::vector<std::string>> FetchSystemImgZipImages(
       system_img_zip_name);
 }
 
+Result<std::unique_ptr<CredentialSource>> GetCredentialSourceHelper(
```

Comment:
nit: `Helper` is a bit vague, what do you think of `CredentialSourceFromFlags`? The main contribution of this function is accessing members of `flags`.